app/views/admin/shared/_product_sub_menu.html.haml

```diff
@@ -0,0 +1,8 @@
+= content_for :sub_menu do
+  %ul#sub_nav.inline-menu
+    = tab :products, match_path: '/products'
+    = tab :option_types, match_path: '/option_types'
+    = tab :properties
+    = tab :prototypes
+    = tab :variant_overrides, url: main_app.admin_inventory_path, match_path: '/inventory'
+    = tab_without_cancan_check :import, url: main_app.admin_product_import_path, match_path: '/product_import'
```

Comment:
An opportunity to improve the (currently random?) order of these entries? Can we move the Inventory entry next to second place after Products? maybe a question that @myriamboure can follow up?

Reply:
I think unless you log in as a superadmin, only Products and Inventory show in that submenu anyway. What's your thinking on changing the order?

Reply:
I think it shouldn't be random, we should group things logically everywhere and seeing products and inventory separated by a random list of menu entries looks wrong.